### DISCUSSION TITLE
怎么关闭邮件通知？

### DISCUSSION THREAD

**Discussion by piwawa** [lobehub/lobehub#2152] [General Question] [answered]
设置自动更新后天天都有action failed邮件，烦的要命，怎么关闭这个邮件通知？没看到 unsubscribed 按钮！ 

![image](https://github.com/lobehub/lobe-chat/assets/46926496/76a73eb4-d36a-415d-a7d7-ad24cc6fca87)

**Comment by arvinxx** [comment, marked as answer]
<img width="661" alt="image" src="https://github.com/lobehub/lobe-chat/assets/134299653/4961153b-30e1-4d0c-a93f-53d3c26368a6">

应该在这里可以关闭的，点ignore就好

**Reply by piwawa** [reply]
谢谢大哥，已设置！
